### DISCUSSION TITLE
Prisma for MongoDB and big app

### DISCUSSION THREAD

**Discussion by trigani** [prisma/prisma#2438] [Q&A] [answered]
Hello,
I need technical advice.
I am currently designing a complex application.
I’m trying to decide what database to use and how to design the relational in the application.
My choice is between MongoDB and PostgreSQL.
The application will have functionality like:
- Trello boards
- CRM
- Invoicing
- ToDo list - Wunderlist
How well he does Prisma whit MongoDB? Trello uses MongoDB.

**Comment by ryands17** [comment, marked as answer]
Hey @trigani :wave: 
A good question indeeed!
As per your requirements, I would suggest to go with Postgres as your DB as the above functionality requires a lot of relational querying and your schema will be organized as per the functionalty and models that you create.

Also, currently Prisma doesn't support MongoDB at the moment, but there's a Feature Request for that which you can track [here](https://github.com/prisma/prisma/issues/1277).

So as it stands, currently I would suggest going with Postgres as it supports lots of features out of the box including raw JSON if you require that.

**Comment by ludralph** [comment]
Hi @trigani,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
